.chloggen/googlemanagedprometheus-metric-prefix.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

```suggestion
change_type: breaking
```
Now that this moves client config, it should be marked as breaking